rotkehlchen/assets/exchanges_mappings/uphold.py

```diff
@@ -33,7 +33,6 @@
     strethaddress_to_identifier('0xd26114cd6EE289AccF82350c8d8487fedB8A0C07'): 'OMG',
     strethaddress_to_identifier('0x4575f41308EC1483f3d399aa9a2826d74Da13Deb'): 'OXT',
     strethaddress_to_identifier('0x408e41876cCCDC0F92210600ef50372656052a38'): 'REN',
-    strethaddress_to_identifier('0xC011a73ee8576Fb46F5E1c5751cA3B9Fe0af2a6F'): 'SNX',
```

Comment:
why remove this?

Reply:
Since all the exchanges needed the same mapping I moved it to common. Common is a mapping that is valid for all the exchanges